dictionaries/en_US/src/en_US.txt

```diff
@@ -16910,6 +16914,7 @@ resurging
 ret
 retailor
 retailors
+retainage
```

Comment:
Please `retainage` to [`dictionaries/en_shared/src/shared-additional-words.txt`](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/en_shared/src/shared-additional-words.txt).